### PR TITLE
feat(react-ui-kit): Add Avatar component

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/react-ui-kit/demo/Demo.jsx [hunk 1]
```diff
@@ -153,6 +155,53 @@ class Demo extends React.PureComponent {
...
...FhEaW1lbnNpb24+CiAgICAgICAgIDxleGlmOlBpeGVsWURpbWVuc2lvbj4xNjwvZXhpZjpQaXhlbFlEaW1lbnNpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgr0f6jpAAAAHGlET1QAAAACAAAAAAAAAAgAAAAoAAAACAAAAAgAAAFq4smoVQAAATZJREFUOBF0kEtLAlEYhp9vxtEudlFp0aZN0CoobFQo6k+0adeynxBtWgr9wJJoEWVBaU6o4DTTXPrOpIiRB85ZnPM97+VIo95I+W8JmAdrlFBunmBXNxAvINUdPw4I73t8P3hI3Z0j4Fgk3RGliyr5020ibwR6JzkbydtYIogfzRFwhNjzWTncYvnSJfJDRCzNo5kSPdM026JzUnNrMxXEtoiDiMLqEus3x8RFG8KECY+plhhEsGwjcPBXQCeGEeXrI9ivkAwDjT3+EJNBWU2vsMX73fOsgJjeHZ+1813yZzvE/S+dHEfPrDW6itiOTef2icHb51TAwAZY3Ntk9apGqDXUSO1+42bWE7jVZvDaI1csIK6poHGSMMYpOJSa2rviQBBPYQUzZ/39j9YL/XZX4QVSFf8BAAD//1tgTEwAAAGDSURBVE2QP0scYRCHn9lb1zv1/siB4pFWNF0ULxdsPMFGmzRBJEXIV9BGCHb5AiEQJIWtfaoUqYTrUoQkKgYCCShYKJyR6O6e++44L2ckU77v/J6ZZ2RutqmEAXR71DYfI0/G0MsUQgHlrpRgIKT79Zju2TnhcAnEPkWQuVZL9Sym/GyK6MU07jKBgoV9eYBTpDpIvHvExY9TsuUJJHUQWNjaZObhjJYmRylvtbhRh+QW8nQEdTlSsfCHn/S2D8laNeKVcaRn77a0L2k+amrt9TyuYWslmZHtVQ2dWZOf/PEX6c4BMhTh6iHJasOz7/Vk8c1LlXYD/ZMghaCv7SdXiySffpO8/47UBg2oaCTEzx+gpQLkpuYVlr680sy85Z+3OVOJSPdOSN59g3LkN7WJFjBIvNYgHy/CjQ3xgIXOhva9rclTzTntWPithUcG+uv6u3i1q5z06RhuqgJ2SH8Hae+t+4vZAMMVA3qfT0m39yGyNf9z9QD560jbddx8Hb3uA24BdMgUmcDgSrIAAAAASUVORK5CYII='
```
Comment (ffflorian) [inline packages/react-ui-kit/demo/Demo.jsx]: What do you think about moving these to files?

File: packages/react-ui-kit/src/Identity/Avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import {filterProps} from '../util';
+
+interface Props<T = HTMLDivElement> extends React.HTMLProps<T> {
+  size: number;
+  forceInitials?: boolean;
+  name: string;
+  base64Image?: string;
+  borderColor?: string;
+  backgroundColor: string;
+  fetchImage?: () => {};
```
Comment (ffflorian) [inline packages/react-ui-kit/src/Identity/Avatar.tsx]: Sorting

File: packages/react-ui-kit/src/Identity/Avatar.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    fontSize,
+    justifyContent: 'center',
+    maxHeight: `${size}px`,
+    maxWidth: `${size}px`,
+    minHeight: `${size}px`,
+    minWidth: `${size}px`,
+  };
+};
+
+const filteredAvatarProps = (props: Object) =>
```
Comment (ffflorian) [inline packages/react-ui-kit/src/Identity/Avatar.tsx]: Shouldn't it be `props: Props`?

File: packages/react-ui-kit/demo/Demo.jsx (hunk 1 above)
Comment (lipis) [reply packages/react-ui-kit/demo/Demo.jsx]: Just for the demo..

File: packages/react-ui-kit/src/Identity/Avatar.tsx (hunk 3 above)
Comment (lipis) [reply packages/react-ui-kit/src/Identity/Avatar.tsx]: Props it is..

**Pull request merged**